.gitlab-ci.yml

```diff
@@ -10,6 +10,7 @@ stages:
 Run tests:
   stage: test
   script:
+    # shouldn't this be run.all ?
```

Comment:
This is bothering me - apparently CI only runs `files-text`, which doesn't even include the vscode keybinding tests.

Reply:
Realised this is for gitlab. Why do we need this file?

Reply:
HM is mirrored to GitLab and is also automatically updated in a NUR there that is triggered by this pipeline. Documentation is also built and hosted on GitLab.

Reply:
I see. So this should be updated to run all the tests?

Reply:
ping :D

Reply:
No, it should stay as-is. We run a small test suite on GitLab just as a sanity check. The full suite is run in the GitHub CI job.